ko/README_ko.md

```
@@ -0,0 +1,332 @@
+
+\| [중국어 간체자] (README.md) |
+
+<!-- LANG_HEADER_END -->
+
+# Ducky SDK 샘플 프로젝트 컬렉션
+
+[! [빌드] (https://github.com/newbe36524/Samples/actions/workflows/build.yml/badge.svg)](https://github.com/newbe36524/Samples/actions/workflows/build.yml)
+[! [.그물] (https://img.shields.io/badge/.NET-10.0-512BD4?logo=dotnet)](https://dotnet.microsoft.com/)
+[! [너겟] (https://img.shields.io/nuget/v/Ducky.Sdk?logo=nuget&label=Ducky.Sdk)](https://www.nuget.org/packages/Ducky.Sdk)
+[! [라이센스] (https://img.shields.io/github/license/newbe36524/Samples?label=License)](LICENSE)
+[! [플랫폼] (https://img.shields.io/badge/Platform-Windows%20%7C%20Linux%20%7C%20macOS-blue)](#)
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)